lib/Profile.dart

```diff
@@ -108,7 +117,7 @@ class _ProfileState extends State<Profile> {
                           child: Text("Joined 03/2020",
                               style: TextStyle(
                                 fontSize: 14,
-                                color: Theme.of(context).accentColor,
+                                color: Color(0xFF848484),
```

Comment:
Is there a specific reason to hardcode this as opposed to putting it in a theme like it was before?

Reply:
On the Figma it's a unique color that isn't part of the theme, I now realize it might have been unintentional since I think this page was made before they added the design system, so I'll check on that!